### PR TITLE
BI 53 - thuhien-issue53: Week 3: Printing invoices

### PR DESCRIPTION
**Pull request by thuhien**
## Exercise

* [Exercise-53](https://github.com/NT-Line-team/Behave_test/issues/<53>)

## Overview

An invoice item has the following main attributes: name, quantity (qty), unit price and amount. 

## Verification Steps

1. Input name of product, qty and price
2. Input y or n to continue

### Docs
print_invoices

### CODE REVIEW

**Comment by thuhien** [comment]
Hi anh @hungphamviet ,
The issue has been fixed by me, please help review code.
Thanks,
Hien Truong

File: python_basic/thuhien/print_invoices.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+information_product = []
+sum_amount = 0
+amount = 0
+total = 0
+vat = 0
+status_invoice = True
+while status_invoice:
+    name_product = raw_input("Name of product: ")
+    qty_product = raw_input("Qty of product: ")
+    price_product = raw_input("Price of product: ")
```
Comment (hungphamviet) [inline python_basic/thuhien/print_invoices.py]: Please use input method for qty and price.

File: python_basic/thuhien/print_invoices.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    information_product.append(name_product)
+    information_product.append(qty_product)
+    information_product.append(price_product)
+    information_product.append(int(qty_product)*int(price_product))
```
Comment (hungphamviet) [inline python_basic/thuhien/print_invoices.py]: No need to cast qty and price once you use `input` method.

File: python_basic/thuhien/print_invoices.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    product_list.append(information_product)
+    information_product=[]
```
Comment (hungphamviet) [inline python_basic/thuhien/print_invoices.py]: Please add spaces around operator.

File: python_basic/thuhien/print_invoices.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    status = raw_input("Add more product(Y/N): ")
+    if status == "n":status_invoice = False
```
Comment (hungphamviet) [inline python_basic/thuhien/print_invoices.py]: Please put if statement and indent block code in separate lines.

File: python_basic/thuhien/print_invoices.py [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+for i in range(len(product_list)):
+    sum_amount += product_list[i][3]
```
Comment (hungphamviet) [inline python_basic/thuhien/print_invoices.py]: Use `for product in product_list` for being better readable.

**Comment by thuhien** [comment]
Hi anh @hungphamviet ,
I have been fixed bugs based on your comments, please help to review again.
Thanks,
Hien Truong

**Pull request merged**